### PR TITLE
bug 104304

### PR DESCRIPTION
**Pull request by quanah**
This moves the Zimbra ClamAV DB into its own package, so that we can update it as necessary with future clamav releases.  This is necessary sometimes as ClamAV periodically changes the format of its signature files.

### CODE REVIEW

**Comment by quanah** [comment]
There may or may not be a dependency between clamav and the underlying database.  Having the database in a specific package means we can later tie a new clamav release to a new version of the database files if required.  And, yes, initial.  These DBs are generally only used on fresh installations.  There has been one specific occasion where a change in the format of the database required a wipe of the existing DBs (that are kept updated by freshclam after these initial ones are deployed).  Having this package allows us to also set an upgrade step in a later version, if necessary, to do something similar.  Given that the ClamAV project just announced they are changing the format (although this time it's backwards compatible), it gives some clue that there will be a release in the near future that utilizes the new format.

**Comment by gordyt** [comment]
+1

File: thirdparty/clamav-db/zimbra-clamav-db/debian/control [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Section: mail
+Priority: optional
+Maintainer: Zimbra Packaging Services <packaging-devel@zimbra.com>
+Build-Depends: debhelper (>= 9)
+Standards-Version: 3.9.5
+
+Package: zimbra-clamav-db
+Architecture: all
+Depends: zimbra-base
+Description: Inital ClamAV Databases for ClamAV
```
Comment (plobbes) [inline thirdparty/clamav-db/zimbra-clamav-db/debian/control]: I'm late to the party, and I see Gren noticed this but it didn't get fixed so I'll comment as well here...
- typo: change 'Inital' to 'Initial' here and on line below.

File: thirdparty/clamav-db/zimbra-clamav-db/rpm/SPECS/clamav-db.spec [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Summary:            Inital ClamAV Databases for ClamAV
```
Comment (plobbes) [inline thirdparty/clamav-db/zimbra-clamav-db/rpm/SPECS/clamav-db.spec]: Same as the other comment - change 'Inital' to 'Initial' here and on line 14 below.

**Pull request merged**